Misc/NEWS.d/next/Documentation/2021-04-14-19-24-39.bpo-43848.gjIRIY.rst

```diff
@@ -0,0 +1 @@
+In documentation of class `GzipFile` in module gzip, explain data type of optional constructor argument `mtime`, and recommend `mtime = 0` for generating deterministic streams.
```

Comment:
Did you want these to be styled as code (`like this` appears in Markdown)? In reStructedText you need two backticks. 
I think that's why `rstlint` is failing.